exercises/twelve-days/twelve-days.spec.js

```diff
@@ -0,0 +1,161 @@
+  });
+
+  test.skip('test sing whole song', () => {
+    const expectedSong =
+      'On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the second day of Christmas my true love gave to me, two Turtle Doves, ' +
+      'and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the third day of Christmas my true love gave to me, three French Hens, two Turtle Doves, ' +
+      'and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the fourth day of Christmas my true love gave to me, four Calling Birds, three French Hens, ' +
+      'two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the fifth day of Christmas my true love gave to me, five Gold Rings, four Calling Birds, ' +
+      'three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, ' +
+      'four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming, ' +
+      'six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, ' +
+      'and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, ' +
+      'seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, ' +
+      'three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking,' +
+      ' seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, ' +
+      'three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing,' +
+      ' eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, ' +
+      'four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, ' +
+      'ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, ' +
+      'six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, ' +
+      'and a Partridge in a Pear Tree.\n' +
+      '\n' +
+      'On the twelfth day of Christmas my true love gave to me, twelve Drummers Drumming, ' +
+      'eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, ' +
+      'seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, ' +
+      'three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
```

Comment:
There are lots of big blocks with `String + String`-- I suggest using some ES6 and making these [multiline strings](https://jack.ofspades.com/multiline-strings-in-es6-javascript), if for no other reason than as a useful example to those who would look at your sample code for inspiration.